data/blog/bitcoin-infrastructure-this-decade.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: "Looking Ahead: Bitcoin Infrastructure This Decade"
+date: '2023-09-20'
+tags: ['OpenSats', 'bitcoin', 'infrastructure', 'research']
+draft: false
+authors: ['niftynei']
+images: ['/static/images/blog/08-nifty-infra.jpg']
+summary: "TODO"
+---
+
+OpenSats aims to be a low-time preference, long-term thinking organization. One good example of this is our recent announcement of the LTS (Long Term Support) program for funding Bitcoin Core contributors with a track record of solid contributions.
+
+As an organization, we aim to focus our funding and efforts on open-source code and education projects that will prove to provide long-term value for Bitcoin as infrastructure. As such, we prioritize funds to builders and experts who are focused on improving the resilience, accessibility, and reach of Bitcoin through important and maybe underfunded work on core technology in the ecosystem.
+
+As infrastructure, it's important that the tech we build supports Bitcoin's core values: self-sovereignty through the preservation of anonymity, accessibility, and decentralization. Together, these values build us a system where trustless financial transactions without intermediaries are a touchable reality for all, not an imagined ideal.
+
+Given these goals, what does a Bitcoin infrastructure look like a decade from now?
+
+- **Mining**, particularly transaction inclusion decisions, and block construction are more distributed, returning true value to the mempool as a necessary gossip protocol for the network.
```

Comment:
**Mining:** Transaction selection and block construction are more distributed???

Reply:
maybe "...construction become more distributed..." instead of "...construction are more distributed..."??